app/views/bulk_update/trainees/add_new_trainees/show.html.erb

```diff
@@ -0,0 +1,80 @@
+<%= render PageTitle::View.new(text: "Bulk add new trainees") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLinkComponent.new(text: "Home", href: root_path) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-xl"><%= organisation_name %></span>
+      Bulk add new trainees
+    </h1>
+
+    <p class="govuk-body">
+      Training providers can register their trainees in bulk by creating trainee records and adding the required data.
+    </p>
+
+    <%= render GovukComponent::InsetTextComponent.new(classes: "") do %>
+      <p class="govuk-body">
+        You must only use this CSV file template to register new trainees.
+        If you use this to update already existing trainees, it will come back
+        as an error.
+      </p>
+    <% end %>
+
+    <h2 class="govuk-heading-m">
+      1. Download this CSV file template
+    </h2>
+
+    <p class="govuk-body">
+      <%= govuk_link_to("Download empty CSV file to add new trainees", "#") %>
+    </p>
+
+    <h3 class="govuk-heading-s">If you're adapting the CSV file template for a direct extract from your student record system</h3>
+
+    <p class="govuk-body">You can adapt this CSV file template to add trainee data directly from your student record system.</p>
+
+    <p class="govuk-body">You must follow the exact order of the data fields in this CSV template, otherwise it will come back as an error.</p>
+
+    <p class="govuk-body">If you're adapting the CSV file template for a direct extract from your student record system, you can go directly to step 3.</p>
+
+    <h2 class="govuk-heading-m">
+      2. Add new trainee details
+    </h2>
+
+    <p class="govuk-body">Open as a CSV file to add trainee details in bulk. There are 36 data fields. You will need to include trainee information such as:</p>
```

Comment:
Just a note that this will probably need to change once we have the final version of the CSV, because the number fields will no doubt be different.
We'll do that in the next ticket.